### ISSUE TITLE
Weird licensing terms

### ISSUE THREAD

**Issue by rimas-kudelis** [extremeshok/clamav-unofficial-sigs#156]
I think the current wording of the license is quite unusual and somewhat unclear. It would be much clearer if instead of stating that "This is property of eXtremeSHOK.com" you would simply add your own copyright statement above or below Bill's. Something like this:

```
Copyright (c) 2015 - 2016, Adrian Jon Kriel :: eXtremeSHOK.com (admin@extremeshok.com)
Copyright (c) 2007 - 2013, Bill Landry (unofficialsigs@gmail.com)

All rights reserved.

<BSD License here>
```

This would make it obvious that the script is still licensed under BSD and would frankly look better than the current wording. :)

**Comment by rimas-kudelis** [comment]
You're closing this without even a reply?

**Comment by extremeshok** [comment]
```
This is property of eXtremeSHOK.com
You are free to use, modify and distribute, however you may not remove this notice.
Copyright (c) Adrian Jon Kriel :: admin@extremeshok.com
```
Thats the license .. as per all the scripts eg.
https://github.com/extremeshok/xshok-proxmox/blob/master/postinstall.sh

**Comment by extremeshok** [comment]
@rimas-kudelis if you have any other non license suggestions for the script, please add them as issues.

**Comment by rimas-kudelis** [comment]
@extremeshok that's the thing: the way you claim your copyright is quite unusual, to say the least. 

Line 1: ```This is property of eXtremeSHOK.com``` could easily be interpreted as you claiming ownership of the scripts on each and every computer the script ends up on.

Line 2 looks like a (somewhat clumsy) duplication of the terms of the BSD license. As such, it is redundant. 

Also, let's not forget that there are a few licenses called BSD (https://en.wikipedia.org/wiki/BSD_licenses), so without the actual license text, or a link to it's particular version, the licensing terms of e.g. `xshok-proxmox` are not 100% clear.

However, I did not file this issue against `xshok-proxmox`. I filed it against `clamav-unofficial-sigs`, which is based on someone else's work, and which already had a proper LICENSE file. I filed it because I believe that it would have been enough (and frankly, would look more professional) to just add a single line claiming your copyright in 2015-2016, instead of adding that unclear/redundant block of text.

Same applies to other files in this repo. It would have been enough to just state that they are distributed under the terms specified in the LICENSE file, instead of marking them as your **property** and somewhat ambiguously referring to the BSD license.
